processing/src/main/java/io/druid/segment/ColumnSelectorBitmapIndexSelector.java

```diff
@@ -111,6 +112,17 @@ public BitmapFactory getBitmapFactory()
   }
 
   @Override
+  public BitmapIndex getBitmapIndex(String dimension)
```

Comment:
I guess I didn't want to touch too many files in this patch (all of the other filters), but I also didn't want to use a different name for this method, since it's the one that really _should_ be called `getBitmapIndex`.

I am indifferent though & happy to go with whatever people think is best